### ISSUE TITLE
缺少常用的参数归一化函数

### ISSUE THREAD

**Issue by AvatarGanymede** [mindspore-ai/mindspore#158]
比如**Pytorch**的`torch.nn.utils.weight_nor,(module, name=weight, dim=0)`，MindSpore就没有，需要开发者自己根据公式定义Cell。
希望官方可以增加常用的参数归一化函数，甚至支持自定义参数归一化函数。
